===================================================================
RCS file: /repository/pecl/event/event.c,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -r1.2 -r1.3
--- pecl/event/event.c	2004/12/13 04:59:14	1.2
+++ pecl/event/event.c	2004/12/13 13:43:58	1.3
@@ -18,7 +18,7 @@
   +----------------------------------------------------------------------+
 */
 
-/* $Id$ */
+/* $Id$ */
 
 #ifdef HAVE_CONFIG_H
 #include "config.h"
@@ -192,7 +192,7 @@
 		for (i = 0; i < n / sizeof(int); i++) {
 			TAILQ_FOREACH(ev, &EVENT_G(signalqueue), ev_signal_next) {
 				if (ev->ev_fd == sig[i]) {
-					event_active(ev, EV_SIGNAL, 1);
+					event_active(ev, EV_SIGNAL, 1 TSRMLS_CC);
 				}
 			}
 		}
@@ -278,7 +278,7 @@
 		fcntl(EVENT_G(signal_pair)[1], F_SETFL, O_NONBLOCK | fcntl(EVENT_G(signal_pair)[1], F_GETFL, 0));
 # endif
 		/* add one end to the event system */
-		event_set(&EVENT_G(sigevent), EVENT_G(signal_pair)[0], EV_PERSIST|EV_READ, signal_cb, &EVENT_G(sigevent));
+		event_set(&EVENT_G(sigevent), EVENT_G(signal_pair)[0], EV_PERSIST|EV_READ, signal_cb, &EVENT_G(sigevent) TSRMLS_CC);
 		event_add(&EVENT_G(sigevent), NULL TSRMLS_CC);
 	}
 #endif
